test/server/test-run-error-formatting-test.js

```diff
@@ -103,7 +103,9 @@ function assertErrorMessage (file, err) {
 
     expect(actual).eql(expectedMsg);
 
-    //NOTE: remove tested messages from list
+    //NOTE: check that list of error types contains error
+    // with this type and remove tested messages from list
+    expect(untestedErrorTypes.indexOf(err.type) !== -1).to.be.ok;
```

Comment:
we get "strange error" because we forgot include type `'actionElementNotIframeError'` in `type.js` https://github.com/DevExpress/testcafe/pull/735/files#diff-9496e797b89e8d22aebbb62b66c64b4aR37